docs/user-guide/global-configurations/manage-notification.md

```diff
@@ -1,5 +1,7 @@
 # Manage Notification
 
+## Introduction [![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/elements/EnterpriseTag.svg)](https://devtron.ai/pricing)
```

Comment:
notification is not enterprise right?

Reply:
It is Enterprise, I guess...
I have never seen it in any OSS setup so far (attaching snapshot)
<img width="1442" alt="image" src="https://github.com/user-attachments/assets/b50848de-9378-4f96-b617-515017fa5f2f">

Reply:
My bad...fixing it...